src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
The class `ColorSupplier` should have an access modifier to follow best practices. Consider adding the `public` access modifier.